src/auth/openWebPage.ts

```diff
@@ -0,0 +1,37 @@
+import { openLoginPrompt } from '../utils/loginPrompt'
+
+interface windowFeatures {
+  width: number
+  height: number
+}
+
+export async function openWebPage(
+  url: string,
+  windowName: string = '',
+  { width, height }: windowFeatures,
```

Comment:
I think it'll be good to provide defaults for `width` and `height` here.

Reply:
fixed.

Reply:
fixed.